dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/OAuth20EndpointServices.java

```diff
@@ -307,14 +315,14 @@ protected void handleEndpointRequest(HttpServletRequest request,
                 break;
             }
         } catch (OidcServerException e) {
-            if (!isBrowser) {
+            if (!isBrowserWithBasicAuth) {
                 // we don't want routine browser auth challenges producing ffdc's.
                 // (but if a login is invalid in that case, we will still get a CWIML4537E from base sec.)
                 // however for non-browsers we want ffdc's like we had before, so generate manually
                 com.ibm.ws.ffdc.FFDCFilter.processException(e,
-                        "com.ibm.ws.security.oauth20.web.OAuth20EndpointServices", "274", this);
+                        "com.ibm.ws.security.oauth20.web.OAuth20EndpointServices", "324", this);
```

Comment:
What's 324?

Reply:
old style ffdc message, it's supposed to be the line number the ffdc is emitted from.  They don't stay correct over time, but I thought I'd fix this one while I was in there.